src/layouts/EditPage/MarkdownEditPage.tsx

```diff
@@ -187,26 +187,33 @@ const PreviewModal = ({
   togglePreview,
 }: PreviewModalProps) => {
   return (
-    <Modal isOpen={isOpen} onClose={onClose} size="xl">
+    <Modal isOpen={isOpen} onClose={onClose} size="4xl" scrollBehavior="inside">
       <ModalOverlay />
       <ModalContent p="1rem">
         <ModalHeader>
-          <Text textStyle="h3">New editing preview</Text>
+          <Text textStyle="h3">Preview Isomer’s new editor</Text>
         </ModalHeader>
         <ModalCloseButton />
```

Comment:
nit: the modal close button is too close to the edge, might need to add the padding similar to how it was done inside CreateMediaFolderModal

Reply:
Fixed in 759bd550!